test/Microsoft.TestPlatform.AcceptanceTests/Microsoft.TestPlatform.AcceptanceTests.csproj

```diff
@@ -7,8 +7,8 @@
   </PropertyGroup>
   <Import Project="$(TestPlatformRoot)scripts/build/TestPlatform.Settings.targets" />
   <PropertyGroup>
-    <OutputType Condition=" $(TargetFramework.StartsWith('netcoreapp')) ">Exe</OutputType>
-    <TargetFrameworks>netcoreapp2.1;net451</TargetFrameworks>
+    <OutputType Condition=" $(TargetFramework.StartsWith('net6')) ">Exe</OutputType>
```

Comment:
Could we find a condition that would also work when we move net6 to net7?

Reply:
I cannot find anything obvious looking online. If you don't have any smart idea, please ignore this.

Reply:
I was thinking the same, but it would be a variation on Not Start With net4, which is even less obvious.